styleguide/assets/mockServiceWorker.js

```diff
@@ -0,0 +1,283 @@
+/**
```

Comment:
We should try to make `styleguidist` serve this directly from `node_modules/msw/lib/iife`